infrastructure/main.tf

```diff
@@ -35,16 +35,30 @@ resource "vault_generic_secret" "servicebus-listen-conn-string" {
     EOT
 }
 
+module "db" {
+  source              = "git@github.com:contino/moj-module-postgres.git"
+  product             = "${var.product}"
+  location            = "${var.location_db}"
+  env                 = "${var.env}"
+  postgresql_database = "letter_tracking"
+  postgresql_user     = "letter_tracking"
+}
+
 module "send-letter-producer-service" {
   source              = "git@github.com:contino/moj-module-webapp?ref=master"
   product             = "${var.product}-producer"
-  location            = "${var.location}"
+  location            = "${var.location_app}"
   env                 = "${var.env}"
   ilbIp               = "${var.ilbIp}"
   resource_group_name = "${azurerm_resource_group.rg.name}"
 
   app_settings = {
```

Comment:
very minor one formatting of the all the variables in this block could be consistent

Reply:
Did you mean  `=` alignment?